src/components/__tests__/page-details.test.jsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-env jest */
+import PageDetails from '../page-details';
+import React from 'react';
+import {shallow} from 'enzyme';
+import simplePage from '../../__mocks__/simple-page.json';
+import WebMonitoringDb from '../../services/web-monitoring-db';
+
+describe('page-details', () => {
+  // Change string values to date objects so they're parsed correctly
+  simplePage.versions.forEach(version => {
+    version.capture_time = new Date(version.capture_time);
+  });
+  const match = { params: { pageId: simplePage.uuid }};
+  const mockApi = Object.assign(Object.create(WebMonitoringDb.prototype), {
+    getPage: jest.fn().mockResolvedValue(simplePage),
+    getVersions: jest.fn().mockResolvedValue(simplePage.versions)
+  });
```

Comment:
Can you wrap this in a function? Technically speaking, it works just fine as-is, but if you made a second test that used `mockApi`, you’d be in trouble: the second test would be using the same mock instance, and you’d have to know which call to ask for in the lines like `await mockApi.getPage.mock.results[n].value`. (Many test frameworks run tests asynchronously or in random order to help make sure you don’t accidentally do this, or accidentally depend on tests that have side-effects.)

It’s safer to create a new mock for each test:

```js
function createMockApi () {
  return Object.assign(Object.create(WebMonitoringDb.prototype), {
    getPage: jest.fn().mockResolvedValue(simplePage),
    getVersions: jest.fn().mockResolvedValue(simplePage.versions)
  });
}

//...and in your test...
it('shows correct title', async () => {
  const mockApi = createMockApi();
  const pageDetails = shallow(<PageDetails match={match} />);
  //...rest of your test with no changes here
});
```